sympy/simplify/fu.py

```diff
@@ -1387,6 +1394,10 @@ def f(rv):
         if not (isinstance(rv, Pow) and isinstance(rv.base, cos)):
             return rv
 
+        e = rv.exp
+        if e % 2 == 1:
+            return TR15(rv.base**(e+1))/rv.base
```

Comment:
```suggestion
            return TR15(Pow(rv.base, e + 1, evaluate=False))/rv.base
```

Reply:
Do you think it would be better to do the following?

```python
    def f(rv):
        if not (isinstance(rv, Pow) and isinstance(rv.base, sin)):
            return rv

        ia = 1/rv
        a = _TR56(ia, sin, cot, lambda x: 1 + x, max=max, pow=pow)
        if a != ia:
            if rv.exp % 2 == 1:
                rv = a*rv.base**-2
            else:
                rv = a
        return rv

    return bottom_up(rv, f)
```

Reply:
I don't think so -- then you would actually be changing the value that is returned. I think what you have is fine. 

But that SO has me wondering about the expression `eq = (7*sin(q) - 9*sqrt(3)*cos(q) - 2*cos(3*q + pi/6))/(4*(2*sin(q)**2 - 5))` and how sympy doesn't simplify to `s = cos(q + pi/6)` but has no problem figuring out that the two are equal.
```python
>>> from sympy.abc import q
>>> eq = (7*sin(q) - 9*sqrt(3)*cos(q) - 2*cos(3*q + pi/6))/(4*(2*sin(q)**2 - 5))
>>> s = cos(q + pi/6)
>>> (eq - s).expand(trig=True).factor()
-sqrt(3)*(sin(q)**2 + cos(q)**2 - 1)*cos(q)/(2*sin(q)**2 - 5)
>>> simplify(_)  # the 2nd term is 0
0
```

Reply:
Interesting.  It's probably that `(sin(q)**2 + cos(q)**2 - 1)` is easy to simplify to 0.  Kind of reminds me of how `x**2+2*x+1` doesn't simplify to `(x+1)**2`, but sympy has no problem figuring out those two are equal by simplifying the difference to 0.

Reply:
Btw I kept what I had and fixed the spacing around the +.